docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -0,0 +1,293 @@
+[role="xpack"]
+[testenv="basic"]
+[[index-lifecycle-management]]
+= Policy Phases and Actions
+
+There are four stages in the index lifecycle:
+
+* Hot--the index is actively being updated and queried.
+* Warm--the index is no longer being updated, but is still being queried.
+* Cold--the index is no longer being updated and is seldom queried. The
+information still needs to be searchable, but it's okay if those queries are
+slower.
+* Delete--the index is no longer needed and can safely be deleted.
+
+Each of these stages is called a "phase". A policy does not need to configure
+each phase for an index. For example, one index may only go from the hot phase to
+the delete phase, while another may run through all phases. Depending on which
+phase one is in, different actions are allowed to be taken.
+
+== Actions
+
+Before discussing which actions are allowed in which phase, it is important
+to understand all the actions as they are.
+
+[[ilm-allocate-action]]
+=== Allocate
+
+The Allocate action allows you to specify which nodes are allowed to host the
+shards of the index and set the number of replicas.
+Behind the scenes, it is modifying the index settings
+for shard filtering and/or replica counts. When updating number of replicas,
```

Comment:
`When updating number` -> `When updating the number`

Reply:
+1